core/src/Kokkos_View.hpp

```diff
@@ -525,10 +525,10 @@ template <class>
 struct is_view : public std::false_type {};
 
 template <class D, class... P>
-struct is_view<View<D, P...> > : public std::true_type {};
+struct is_view<View<D, P...>> : public std::true_type {};
 
 template <class D, class... P>
-struct is_view<const View<D, P...> > : public std::true_type {};
```

Comment:
It's somewhat surprising that both versions passed the indentation check.

Reply:
I had the same thought.